data/lib/core/bestiary.lua

```diff
@@ -0,0 +1,34 @@
+local bestiaryData = {
+	-- dummy tables for very rare and none
+	[-2] = {kills = {0, 0, 0}},
+	[-1] = {kills = {2, 3, 5}},
+
+	-- number of stars (from harmless to challenging)
+	-- rarity 4 triggers rare values
+	[0] = {kills = {5, 10, 25},  charmPoints = 1},
+	[1] = {kills = {10, 100, 250}, charmPoints = 5},
+	[2] = {kills = {25, 250, 500},  charmPoints = 15},
+	[3] = {kills = {50, 500, 1000}, charmPoints = 25},
+	[4] = {kills = {100, 1000, 2500}, charmPoints = 50},
+	[5] = {kills = {200, 2000, 5000}, charmPoints = 100}
+}
```

Comment:
What if I want to create a monster with a custom amount of kills per level?

I'm OK with defaulting to Tibia behavior, but not as much with only being able to have Tibia behavior.

Reply:
I don't know if the one [I posted above](https://github.com/otland/forgottenserver/pull/4412#issuecomment-1517881166) is enough.